packages/fluent/scss/listview/_variables.scss

```
@@ -0,0 +1,93 @@
+@use "../core" as *;
+@use "../list/_variables.scss"as *;
+
+// Listview
+/// Horizontal padding of listview.
+/// @group listview
+$kendo-listview-padding-x: map-get( $kendo-spacing, 1 ) !default;
+/// Vertical padding of listview.
+/// @group listview
+$kendo-listview-padding-y: map-get( $kendo-spacing, 1 ) !default;
+
+/// Width of listview.
+/// @group listview
+$kendo-listview-border-width: map-get( $kendo-spacing, hair ) !default;
+
+/// Horizontal padding of listview header.
+/// @group listview
+$kendo-listview-header-padding-x: map-get( $kendo-spacing, 4 ) !default;
+/// Vertical padding of listview header.
+/// @group listview
+$kendo-listview-header-padding-y: map-get( $kendo-spacing, 2 ) !default;
+
+/// Horizontal padding of listview footer.
+/// @group listview
+$kendo-listview-footer-padding-x: $kendo-listview-header-padding-x !default;
+/// Vertical padding of listview footer.
+/// @group listview
+$kendo-listview-footer-padding-y: $kendo-listview-header-padding-y !default;
+
+/// Horizontal padding of listview items.
+/// @group listview
+$kendo-listview-item-padding-x: map-get( $kendo-spacing, 1 ) !default;
+/// Vertical padding of listview items.
+/// @group listview
+$kendo-listview-item-padding-y: map-get( $kendo-spacing, 1 ) !default;
+
+/// Font family of listview.
+/// @group listview
+$kendo-listview-font-family: var( --kendo-font-family, initial ) !default;
+/// Font size of listview.
+/// @group listview
+$kendo-listview-font-size: var( --kendo-font-size, initial ) !default;
+/// Line height of listview.
+/// @group listview
+$kendo-listview-line-height: var( --kendo-line-height, initial ) !default;
+
+/// Gap between items of grid listview.
+/// @group listview
+$kendo-listview-grid-gap: map-get( $kendo-spacing, 0 ) !default;
+
+/// Text color of listview.
+/// @group listview
+$kendo-listview-text: var( --kendo-component-text, initial ) !default;
+/// Background color of listview.
+/// @group listview
+$kendo-listview-bg: var( --kendo-component-bg, initial ) !default;
+/// Border color of listview.
+/// @group listview
+$kendo-listview-border: var( --kendo-component-border, initial ) !default;
+
+/// Text color of focused listview items.
+/// @group listview
+$kendo-listview-item-focus-text: initial !default;
+/// Background color of focused listview items.
+/// @group listview
+$kendo-listview-item-focus-bg: initial !default;
+/// Box shadow of focused listview items.
+/// @group listview
+$kendo-listview-item-focus-shadow: var( --kendo-list-item-focus-shadow, $kendo-list-item-focus-shadow ) !default;
+
+/// Text color of selected listview items.
+/// @group listview
+$kendo-listview-item-selected-text: initial !default;
+/// Background color of selected listview items.
+/// @group listview
+$kendo-listview-item-selected-bg: rgba( get-theme-color( neutral, 40 ), .25 ) !default;
```

Use a value from the primary pallet with a lower hue e.g:

```suggestions
$kendo-listview-item-selected-bg: get-theme-color-var( neutral-20 ) !default;
```